src/linear_algebra/matrix/to_lin.lean

```diff
@@ -134,7 +241,7 @@ by { ext, rw [matrix.one_apply, linear_map.to_matrix'_apply, id_apply] }
 
 @[simp] lemma matrix.to_lin'_mul [fintype m] [decidable_eq m] (M : matrix l m R)
   (N : matrix m n R) : matrix.to_lin' (M ⬝ N) = (matrix.to_lin' M).comp (matrix.to_lin' N) :=
-by { ext, simp }
+by { ext, simp, }
```

Comment:
I don't have a strong opinion on whether to end tactic blocks with an extra comma, but this change is too annoying for reviewing or `git blame`ing, IMO.
```suggestion
by { ext, simp }
```